lib/sedna/service/server/aggregation.py

```diff
@@ -13,26 +13,28 @@
 # limitations under the License.
 
 import time
-from typing import List, Optional, Dict, Any
-
 import uuid
-from pydantic import BaseModel
+from typing import Any, Dict, List, Optional
+
 from fastapi import FastAPI, WebSocket
 from fastapi.routing import APIRoute
+from plato.config import Config
```

Comment:
suggest: As plato is not used as a basic dependency, it is recommended to move it to `__init__` of `AggregationServerV2`

Reply:
updated